webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const webpack = require('webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const buildDir = __dirname + '/lib';
+
+const localConfig = {
+    js_filename: '[name].min.js'
+};
+
+module.exports = environment => {
+    return {
+        entry: {
+            highlight: './src/resources/js/highlight'
+        },
+        output: {
+            path: path.resolve(buildDir),
+            filename: localConfig['js_filename']
+        },
+        mode: 'production',
+        module: {
+            rules: [
+                {
+                    test: /\.js$/,
+                    use: {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: ['@babel/preset-env']
+                        }
+                    }
+                }
+            ]
+        },
+        optimization: {
+            minimize: false,
+            minimizer: [new TerserPlugin({
+                extractComments: false,
+            })],
+        },
+        plugins: [
+            new CopyWebpackPlugin({
+                patterns: [
+                    {
+                        from: path.resolve(__dirname + '/node_modules/pako/dist/pako_inflate.js'),
+                        to: '[name].min[ext]'
+                    }
+                ]
+            })
+        ]
+    }
+};
```

Comment:
Phew, I have to admit that I can't really understand what's happening here... But that's because I'm not really familiar with webpack either 😉

I'm sure this is all correct, but as an ignoramus I find it strange that there is an `entry` for hljs but none for pako. On the other hand, there is a special `CopyWebpackPlugin` activity for pako, but not for hljs... hm 🤔

Is it possibly that the `entry` instructs webpack to build/bundle hljs as defined in our new highlight.js?
And with pako is it necessary to simply copy the inflate file? And no rebundling?

Reply:
That's indeed what it does